fdbclient/ManagementAPI.actor.cpp

```diff
@@ -1258,93 +1257,152 @@ struct AutoQuorumChange : IQuorumChange {
 };
 Reference<IQuorumChange> autoQuorumChange( int desired ) { return Reference<IQuorumChange>(new AutoQuorumChange(desired)); }
 
-ACTOR Future<Void> excludeServers(Database cx, vector<AddressExclusion> servers, bool failed) {
-	state Transaction tr(cx);
-	state Key versionKey = BinaryWriter::toValue(deterministicRandom()->randomUniqueID(),Unversioned());
-	state std::string excludeVersionKey = deterministicRandom()->randomUniqueID().toString();
+void excludeServers(Transaction& tr, vector<AddressExclusion>& servers, bool failed) {
```

Comment:
Conventionally we take a pointer to transaction here, but it might be good to keep it consistent.

Reply:
Yeah, a pointer should be better. I use reference for consistency.